assemblyline/common/tag_whitelist.yml

```diff
@@ -19,9 +19,21 @@ match:
   # Direct match to dynamic domains (localhost)
```

Comment:
(localhost) no longer apply, just remove that,  same below

Reply:
Addressed in 10bc286fa6d819aaef9c190e57a5cecf71294139